### ISSUE TITLE
Commission or Fees before creating product

### ISSUE THREAD

**Issue by dmsmatrix** [allegro/allegro-api#9460]
Hi,

Is there any API to check commission fees on particular leaf level category before ow without creating product.

I can see an API called **Calculate fee and commission for an offer** with endpoint: https://developer.allegro.pl/documentation#operation/calculateFeePreviewUsingPOST, but its after creating offer.

Thanks in advance..

**Comment by Lukasz-Zurek** [comment]
> Is there any API to check commission fees on particular leaf level category before ow without creating product.
> 
> I can see an API called **Calculate fee and commission for an offer** with endpoint: https://developer.allegro.pl/documentation#operation/calculateFeePreviewUsingPOST, but its after creating offer.

Not really, you do not have to create an offer first. Obviously while using [GET /billing/billing-entries](https://developer.allegro.pl/documentation/#operation/getBillingEntries), we require a full offer data in the offer structure. So you can retrieve this data via [GET /sale/offers/{offerId}](https://developer.allegro.pl/documentation/#operation/getOfferUsingGET). For example - you can create a draft and check commission fees based on the offer structure. But still we need this data and we do not have any other resource to calculate it.
